lib/commands/context.js

```diff
@@ -329,6 +332,18 @@ async function getChromedriverPort (portSpec) {
   return foundPort;
 }
 
+helpers.isChromedriverAutodownloadEnabled = function isChromedriverAutodownloadEnabled () {
```

Comment:
I think it should be `isChromedriverAutoDownloadEnabled` but it doesn't matter. I just wanted to comment in case it was a typo